PERFORMANCE.md

```diff
@@ -146,3 +146,15 @@ new ProduceRecordTransformationFilter(
 10000000 records sent, 60679.611650 records/sec (59.26 MB/sec), 25.00 ms avg latency, 1080.00 ms max latency, 1 ms 50th, 170 ms 95th, 601 ms 99th, 992 ms 99.9th.
 
 ```
+
+### Jenkins pipeline for performance
+When a PR is created and the performance tests are needed, the following comment shall be added into the PR:
```

Comment:
We should probably mention that this is restricted to specific people.

Reply:
+1 @franvila could you suggest some words.

Reply:
We could say something like "Only available for kroxylicious developers"

Reply:
@franvila how does that look?

Reply:
I see you've looked!